csrc/id_model/indexing.cpp

```diff
@@ -303,6 +305,22 @@ void IdGraphIndexCompute::handle(Merge* merge) {
   }
 }
 
+void IdGraphIndexCompute::handle(Swizzle* swizzle) {
+  const bool is_forward = isForward(swizzle);
+
+  if (is_forward) {
+    auto x_idx = getIndex(swizzle->inX());
+    auto y_idx = getIndex(swizzle->inY());
+    setIndex(swizzle->outX(), x_idx);
+    setIndex(swizzle->outY(), y_idx);
+  } else {
+    auto x_idx = getIndex(swizzle->outX());
+    auto y_idx = getIndex(swizzle->outY());
+    setIndex(swizzle->inX(), x_idx);
+    setIndex(swizzle->inY(), y_idx);
+  }
```

Comment:
Oh, just realized that swizzle is lowered as a no-op, but still, all tests pass... Seems to be because when we use tensor indexer to index gmem tensor of TMA, the indices are always 0 for swizzle... Will leave this PR as is, and write a separate PR to fix it.